### ISSUE TITLE
Staging Review finding: "Show all times" button likely to be missed by screen magnification users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#97471]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** CMS Team
**Product:** Next Build and Next Build's Events and Event Listing Templates
**Feature:** Events and Event Listing Templates

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On an event page with other times for the event ([example](https://staging.va.gov/outreach-and-events/events/64661/)), in the "View other times" accordion, a "Show all times" button causes more events to be added to the accordion.

The "Show all times" button is placed in the far right bottom corner. For a low vision user who uses a screen magnifier (as opposed to browser zoom), we would expect that most users would follow the main column of content down along the left, moving right across the page only when there's reason to believe there's more content, eg. following text or following a design element suggestive of more content. The "Show all times" button is off floating on its own in a spot that doesn't have anything pointing users toward it, and is likely to be missed.

## Recommended action

I recommend aligning the "Show all times" button to the left. I'm not sure what benefit there is to floating it right, and a left alignment should avoid this issue.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen magnification
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
